src/redismodule.h

```diff
@@ -506,6 +507,7 @@ typedef struct RedisModuleTypeMethods {
     RedisModuleTypeAuxLoadFunc aux_load;
     RedisModuleTypeAuxSaveFunc aux_save;
     int aux_save_triggers;
+    RedisModuleTypeFreeEffortFunc free_effort;
```

Comment:
We'll require to bump the version number here to maintain ABI compatibility, as modules compiled with an older `redismodule.h` will have a smaller `RedisModuleTypeMethods` and may end up with a junk pointer.

Reply:
Ok, thank you for your review. I added a v3 version of API, which included free_effort.

Reply:
Ok, thank you for your review. I added a v3 version of API, which included free_effort.